extensions/amp-ad-exit/0.1/config.js

```diff
@@ -14,8 +14,12 @@
  * limitations under the License.
  */
 
-import {user} from '../../../src/log';
+import {user, dev} from '../../../src/log';
 import {FilterType} from './filters/filter';
+import {ANALYTICS_CONFIG} from '../../amp-analytics/0.1/vendors';
```

Comment:
Nevermind, found a small number of precedents such as https://github.com/ampproject/amphtml/blob/5f69c782b6e975c6dedf7b6e93037f9aeff40376/extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js#L23